modules/stochastic_tools/doc/content/modules/stochastic_tools/examples/bayesian_uq_1d_diff.md

```diff
@@ -0,0 +1,207 @@
+
+!listing modules/stochastic_tools/test/tests/samplers/mcmc/main_base.i block=Samplers
+
+5. In the [MultiApps](MultiApps/index.md) block, a [SamplerFullSolveMultiApp](/SamplerFullSolveMultiApp.md) object is defined to create and run a sub-application for each sample provided by the sampler object. More specifically, a full-solve type sub-application is executed for each row of the Sampler matrix.
+
+!listing modules/stochastic_tools/test/tests/samplers/mcmc/main_base.i block=MultiApps
+
+6. Under the [Transfers](Transfers/index.md) block, [SamplerReporterTransfer](/SamplerReporterTransfer.md) is utilized to transfer data from [Reporters](Reporters/index.md) on the sub-aplication to a [StochasticReporter](/StochasticReporter.md) on the main application. In this example, the value of a postprocessor named `average`, as defined in the sub-application, is transferred to a [StochasticReporter](/StochasticReporter.md) named `constant` in the master application.
+
+!listing modules/stochastic_tools/test/tests/samplers/mcmc/main_base.i block=Transfers
+
+7. The [Controls](Controls/index.md) block makes use of [MultiAppSamplerControl](MultiAppSamplerControl.md) to pass command line arguments from the master application to the sub-applications. In this example, three arguments are passed from the master application to the sub-spplications, i.e.,
+
+- `left_bc`: the Dirichlet boundary condition at the lefthandside of the domain. 
+- `right_bc`: the Dirichlet boundary condition at the lefthandside of the domain.
+- `mesh1`: the domain length, which is controled by the configuration parameters in the [Samplers](Samplers/index.md) block.
+
+!listing modules/stochastic_tools/test/tests/samplers/mcmc/main_base.i block=Controls
+
+8. Then, the [Reporters](Reporters/index.md) section defines `Reporter` objects for data transfer in the MultiApps system. 
+
+- The `constant` reporter of type [StochasticReporter](/StochasticReporter.md) serves as a storage container for stochastic simulation results coming from the sub-application. 
+
+- `noise_specified` makes use of the [ConstantReporter](/ConstantReporter.md) reporter to generate a constant value, which is used for the variance term in the experimental error in this example. The value is specified to be 0.05 as defined in `real_values`. 
+
+- `mcmc_reporter` is of type [PMCMCDecision](/PMCMCDecision.md) which makes the decision whether or not to accept a proposal generated by the parallel Markov chain Monte Carlo algorithms. It takes in a likelihood function `likelihoods`, a `sampler`, and the model output from the sub-application `output_value`. In this example, `output_value = constant/reporter_transfer:average:value` means that the model output from the sub-application is obtained by a reporter named `constant`, which interacts with the reporter `reporter_transfer` to transfer the average solution field.
```

Comment:
PMCMCDecision does not perform actual decision making. It needs to be a specific MCMC decision making class. Please change the .i file being used. Please use preferably, main_des.i